mlflow/R/mlflow/R/tracking-runs.R

```diff
@@ -275,8 +275,9 @@ mlflow_search_runs <- function(filter = NULL,
     run_view_type = run_view_type
   ))
 
-  response$run %>%
-    purrr::map_df(parse_run)
+  runs_list <- response$run %>%
+    purrr::map(parse_run)
+  do.call("rbind", runs_list) %||% data.frame()
```

Comment:
Nice. 

I wonder if there is any difference in performance. Maybe we should put a comment to add the dplyr back after 1.0 and update minimum rlang version to 0.3.4?

Reply:
Yeah my guess is that this approach is probably slower. Will add the comment